src/core/types/base/logging.ts

```diff
@@ -0,0 +1,7 @@
+export enum LogType {
+	All = "ALL",
```

Comment:
what is 'all'? its is processed as being 'log' 
also i'd prefer "stdout" or "console" for the default logging (i guess you called this "log")

Reply:
The default goes to all loggers including console.  Here's a rundown of each log type:

All = "ALL", log to all available loggers including console
Log = "LOG", log to console only
External = "EXTERNAL", log to only external loggers (slack, discord), if available
Discord = "DISCORD", log to only discord, if available
Slack = "SLACK", log to only slack, if available

Maybe LogType is the incorrect naming for that enum?

Reply:
Nah I think type is fine for now.
Maybe rename "LOG" to "CONSOLE" or "STDOUT". Rename "EXTERNAL" to "EXTERNALS" plural. Besides that im down and agree with this PR!